src/core/ZkitTSGenerator.ts

```diff
@@ -93,6 +94,7 @@ export default class ZkitTSGenerator extends BaseTSGenerator {
     const templateParams: WrapperTemplateParams = {
       circuitClassName: this._getCircuitName(circuitArtifact),
       publicInputsInterfaceName: this._getInterfaceName(circuitArtifact, "Public"),
```

Comment:
```suggestion
      publicInputsTypeName: this._getInterfaceName(circuitArtifact, "Public"),
```

And the others as well please